students/14th/team_1/seungje/project_westagram/project_westagram/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path,include
+
+urlpatterns = [
+    path('account/',include('account.urls'))
+
```

Comment:
```suggestion
    path('account',include('account.urls'))
```
main url에는 슬래시('/')를 붙이지 않고, app url의 앞부분에 슬래시를 붙이시는 것이 위코드 규칙입니다 :)